src/main/java/com/ideal/studentlog/config/CORSFilter.java

```diff
@@ -0,0 +1,38 @@
+package com.ideal.studentlog.config;
+
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@Component
+@Order(Ordered.HIGHEST_PRECEDENCE)
```

Comment:
What happens if we remove this line?

Reply:
Can access the APIs from next.js without any issue even after removing the line